.github/ISSUE_TEMPLATE/improvement.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Improvement
+about: Suggest an idea/ improvement for this project
```

Comment:
✏️ I'd remove that space between the `/` and `improvement` or change from using a slash to using the word `or`.